multi-tenancy/schema-isolation/standalone.xml

```diff
@@ -542,70 +550,71 @@
                         <property name="isAutoSchemaUpdate">true</property>
                         <property name="authorizationEnabled">true</property>
                         <property name="jobExecutorDeploymentAware">true</property>
+                        <property name="historyCleanupBatchWindowStartTime">00:01</property>
```

Comment:
Let's keep the Camunda subsystem as-is, if there is no need to adjust it due to changes in the engine itself.

Reply:
This also applies to the other changes to the subsystem below this one here

Reply:
If you agree, we can also revert the changes to that subsystem in the other `standalone.xml`